slides/how_to_redbrick.html

```diff
@@ -0,0 +1,204 @@
+      document.getElementsByTagName( 'head' )[0].appendChild( link );
+    </script>
+  </head>
+  <body data-bg-text="Pictures are being taken to be used on social media">
+    <div class="reveal">
+      <div class="slides">
+        <section data-markdown>
+		<script type='text/template'>
+                # How to Redbrick
+                RC Clients, Multiplexers, Web Hosting, Development
+		</script>
+        </section>
+        <section data-markdown>
+                <script type='text/template'>
+                ## Login
+                How to ssh
+                </script>
+                </section>
+        <section data-markdown>
+                <script type='text/template'>
+                ## Type this in your terminal
+                        ~$ ssh Username@pyg.redbrick.dcu.ie
+                </script>
+                </section>
+        <section data-markdown>
+                <script type='text/template'>
+                ![login screen](./images/MainHome.png)
+                </script>
+                </section>
+        <section data-markdown>
+        	<script type='text/template'>
+                ## Multiplexer
+                Tmux
+		</script>
+		</section>
+        <section data-markdown>
+                <script type='text/template'>
+                ### Tmux
+                - Lighter than screen (which <strike>is shit</strike> we don't use)
+                - Better plugins
+                - Scriptable
+                - To set it up
+                 - **~edu/setupTmux**
+                 - rebinds keys ctrl+b > ctrl+a    
+                 - makes it pretty 
+                </script>
+                </section>
+        <section data-markdown>
+                <script type='text/template'>
+                ### Auto Attach
+                - nano ~/.zlogin
+                        tmux attach -d
+                </script>
+                </section>
+        <section data-markdown>
+                <script type='text/template'>
+                ## IRC Clients
+                irssi or weechat 
+                </script>
+                </section>
+        <section data-markdown>
+                <script type='text/template'>
+                ### Weechat
+                - Alternative irc client
+                - Some advantages over irrsi
+                - Looks nicer
+                - Easier to install scripts
+                </script>
+                </section>
+        <section data-markdown>
+                <script type='text/template'>
+                ##### Need to compile it from source, but there's a script for that
+                        ~$ ~/edu/compileweechat
+                </script>
+                </section>        
+        <section data-markdown>
+                <script type='text/template'>
+                ##### We need to set our NICK for the server
+                       ~$ nano ~/.weechat/irc.conf
```

Comment:
https://github.com/redbrick/eduScripts/blob/master/compileweechat#L14
line 14-18 is all you need now